sdk/eventhub/perf-tests.yml

```diff
@@ -0,0 +1,28 @@
+Service: eventhubs
+
+Project: sdk/eventhub/azure-eventhub
+
+PrimaryPackage: azure-eventhub
+
+PackageVersions:
+- azure-core: 1.26.1
+  azure-eventhub: 5.11.1
```

Comment:
will these values have to be updated every time we release a new version or will it continue to work how it does today by picking the latest version in pypi & main using the regex?

Reply:
Currently, these need to be updated manually, but it's debatable whether this is a feature or a bug.  We update the baselines versions after we verify there is no regression in main.

I have an issue open to add a special version "latest-ga" which would automatically use the latest GA version from package manager, however I am hesitant to do this because it might hide regressions if the baseline is updated after a regression we missed.

https://github.com/Azure/azure-sdk-tools/issues/3553